### PR TITLE
Reorder names

### PR DESCRIPTION
**Pull request by michaelchacko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ See 'lib' folder for the specific files.
 </p>
 
 ## Authors
-* Ishan Mitra
+* Michael Chacko
```
Comment (ishan22) [inline README.md]: Not a real person please remove

Comment (chackotaco) [inline README.md]: </3

**Comment by ishan22** [comment]
Michael is useless :smile:

**Comment by ishan22** [comment]
No.

**Pull request closed without merging**